tests/foreman/cli/test_reporttemplates.py

```diff
@@ -63,878 +63,908 @@
-
-        assert host1['name'] in [item.split(',')[0] for item in result_no_filter]
-        assert host2['name'] in [item.split(',')[0] for item in result_no_filter]
-
-        result = ReportTemplate.generate(
-            {
-                'name': rt_host_statuses['name'],
-                'inputs': (
-                    rt_host_statuses['template-inputs'][0]['name']
-                    + "="
-                    + 'name={}'.format(host1['name'])
-                ),
-            }
-        )
-        assert host1['name'] in [item.split(',')[0] for item in result]
-        assert host2['name'] not in [item.split(',')[0] for item in result]
+@pytest.fixture(scope='module')
+def local_subscription(local_org, local_ak):
+    subscription = Subscription.list({'organization-id': local_org['id']}, per_page=False)[0]
+    ActivationKey.add_subscription({'id': local_ak['id'], 'subscription-id': subscription['id']})
 
-    @pytest.mark.tier2
-    def test_positive_lock_and_unlock_report(self):
-        """Lock and unlock report template
+    return subscription
 
-        :id: df306515-8798-4ce3-9430-6bc3bf9b9b33
 
-        :setup: User with reporting access rights, some report template that is not locked
+@pytest.mark.tier2
+def test_positive_report_help():
+    """hammer level of help included in test:
+     Base level hammer help includes report-templates,
+     Command level hammer help contains usage details,
+     Subcommand level hammer help contains usage details
 
-        :steps:
+    :id: ec395f47-a55f-441a-9cc6-e49400c83e8e
 
-            1. hammer report-template update ... --locked true
-            2. hammer report-template update ... --locked false
+    :setup: Any satellite user
 
-        :expectedresults: Report is locked and unlocked successfully.
+    :steps:
 
-        :CaseImportance: Medium
-        """
-        name = gen_string('alpha')
-        report_template = make_report_template({'name': name})
-        ReportTemplate.update({'name': report_template['name'], 'locked': 1})
-        new_name = gen_string('alpha')
-        with pytest.raises(CLIReturnCodeError):
-            ReportTemplate.update({'name': report_template['name'], 'new-name': new_name})
+        1. hammer --help
+        2. hammer report-template --help
+        3. hammer report-template create --help
 
-        ReportTemplate.update({'name': report_template['name'], 'locked': 0})
-        result = ReportTemplate.update({'name': report_template['name'], 'new-name': new_name})
-        assert result[0]['name'] == new_name
+    :expectedresults: report-templates command is included in help,
+                      report-templates command details are displayed,
+                      report-templates create command details are displayed
 
-    @pytest.mark.tier2
-    def test_positive_report_add_userinput(self):
-        """Add user input to template
+    """
+    command_output = '\n'.join(Base().execute('--help'))
```

Comment:
I rewrote the assertion pattern in this test, the old one really bugged me. Before it was generating a list of strings by iterating over the command output multiple times, and then asserting that the length of the generated list of strings was greater than one. A very round-about way of asserting some strings are in another list of strings.